evap/results/templatetags/results_templatetags.py

```diff
@@ -19,3 +19,8 @@ def evaluation_results_cache_timeout(evaluation):
     if evaluation.state == 'published':
         return None  # cache forever
     return 0  # don't cache at all
+
+
+@register.filter(name='participationclass')
+def participationclass(number_of_voters, number_of_participants):
+    return round((number_of_voters/float(number_of_participants))*10)
```

Comment:
why the float? Python3 int division should return floats.

Reply:
old habit